support-workers/src/main/scala/com/gu/stripe/StripeService.scala

```diff
@@ -48,4 +42,10 @@ class StripeServiceForCurrency(stripeService: StripeService, currency: Currency)
   private def getAuthorizationHeader() =
     Map("Authorization" -> s"Bearer ${config.forCurrency(Some(currency)).secretKey}")
 
-}
\ No newline at end of file
+  val createCustomerFromPaymentMethod = com.gu.stripe.createCustomerFromPaymentMethod.apply(this)_
```

Comment:
not totally happy with this but it does keep the `stripeService.something(asdf)` syntax without making stripeService become non cohesive.  Hopefully someone can suggest a nicer way to avoid the more explicit but less auto complete friendly `Something(stripeService, asdf)`?  Or just say it's fine to go that way!

Reply:
another option would be implicit class (extension method), but then you have to get the imports and implicits are usually a last resort

Reply:
I think this is a good approach